src/styles/blocks/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  box-sizing: border-box;
```

Comment:
You should avoid using `box-sizing: border-box;` directly in the element. It's better to apply it globally to every element using the `*` css selector. This is a Code Style issue.